### PR TITLE
add api to get timezones

### PR DESCRIPTION
**Pull request by prakashchoudhary07**
Api details

---
Route: `/api/v1/timezone?userTz=Asia/Kolkata`
---
Response: 

```

{
data:
  timeZones: [{
    label: "Alofi - Niue Time (GMT -11:00)"
    value: "Pacific/Niue"
  },
  ...
  ]
  userTimeZoneIndex: 242
}



```

### CODE REVIEW

**Review by sakshambhatt — APPROVED** [review]
Everything seems good to me :+1:

File: routes/index.ts [hunk 1]
```diff
@@ -12,5 +13,6 @@ app.use('/health', health);
 app.use('/users', users);
 app.use('/calendar', calendar);
 app.use('/events', events);
+app.use('/timezone', timezone);
```
Comment (isVivek99) [inline routes/index.ts:16]: why are we creating seperate route, can we not add to existing route?

**Review by harshith-venkatesh — APPROVED** [review]

File: routes/index.ts (hunk 1 above)
Comment (isVivek99) [reply routes/index.ts:16]: i think we can have it under users, since it comes under userOnboarding

Comment (sakshambhatt) [reply routes/index.ts:16]: /timezone for now. Will review later if needed

**Pull request merged**